src/display/node_stream.js

```diff
@@ -30,6 +30,10 @@ const fileUriRegex = /^file:\/\/\/[a-zA-Z]:\//;
 
 class PDFNodeStream {
   constructor(source) {
+    // prepend 'file:///' to windows absolute path
```

Comment:
"Prepend" and "Windows" should be capitalized, and the sentence should end with a period.